config/common/errors.go

```diff
@@ -48,6 +48,9 @@ var (
 	// systemd
 	ErrTooManySystemdSources = errors.New("only one of the following can be set: contents, contents_local")
 
+	// mount points
+	ErrMountPointForbidden = errors.New("mounts must be under /etc or /var")
```

Comment:
Due to https://github.com/coreos/butane/pull/449#discussion_r1191447738, this is actually a mount unit error after all, so let's move it back.  :flushed:

And we should probably clarify it a bit.  For consistency with the others, `path must be under /etc or /var if with_mount_unit is true`?

Reply:
no problem, changes pushed!